modules/scalacheck/src/main/scala/GenModule.scala

```diff
@@ -4,6 +4,37 @@ package schema
 
 package scalacheck
 
+import org.scalacheck.Gen
+
 trait GenModule extends SchemaModule {
-  // def gen[A](schema: Schema[A])(implicit arb: Arbitrary[Prim[_]]): Gen[A]
+
+  def toGen: Schema ~> Gen = new (Schema ~> Gen) {
+    override def apply[A](schema: Schema[A]): Gen[A] = schema match {
+      case _: Schema.PrimSchema[_]        => ???
```

Comment:
I'm not sure how could I convert `arb: Arbitrary[Prim[_]]` to generator of type `Gen[A]`.
In [xenomorph](https://github.com/nuttycom/xenomorph/blob/master/modules/scalacheck/src/main/scala/xenomorph/scalacheck/ToGen.scala#L34) typeclass `ToGen` exists that is used to convert `Prim[A]` to `Gen[A]`, @vil1  do you want me to implement something similar here?